kong/cmd/utils/nginx_signals.lua

```diff
@@ -84,13 +84,24 @@ function _M.start(kong_conf)
 
   log.debug("starting nginx: %s", cmd)
 
-  local ok, _, _, stderr = pl_utils.executeex(cmd)
-  if not ok then
-    return nil, stderr
+  if kong_conf.nginx_daemon == "on" then
+    -- running as daemon, so we capture command output to temp files using
+    -- the "executeex" method
+    local ok, _, _, stderr = pl_utils.executeex(cmd)
```

Comment:
Just below this line, should print the `stdout` of nginx? Either here or in the `not ok` path. The rationale would be to give a more "native" feeling to the user (with `print`, not with `log`), so the output looks barefeet from nginx itself. What do you think?

Reply:
Probably not - if we're running in daemon mode, let's stay consistent with today's behavior, little value in this.